package.json

```diff
@@ -6,12 +6,20 @@
     "build": "rm -rf ./lib & tsc",
     "build:example": "webpack --config ./example/webpack.prod.cjs",
     "dev": "webpack-dev-server --config ./example/webpack.config.cjs --hot",
-    "test": "jest",
+    "test": "jest src",
     "lint": "eslint src/**/* example/src/**/* --ext .js,.ts,.tsx",
     "typecheck": "tsc --noEmit"
   },
+  "babel": {
+    "presets": [
+      "@babel/preset-typescript"
+    ]
+  },
```

Comment:
`esbuild-jest` unable to transform imported types, this is an alternative solution for now. https://github.com/aelbore/esbuild-jest/issues/57